msteams-platform/apps-in-teams-meetings/build-apps-for-teams-meeting.md

```diff
@@ -0,0 +1,205 @@
+
+## App manifest settings for apps in meeting stage
+
+To share an app to the meeting stage, the app must call the meeting side panel experience in manifest setting:
+
+```json
+"context":[ 
+    "meetingSidePanel", 
+    "meetingStage" 
+     ] 
+```
+
+## Advanced share to stage APIs
+
+There are many scenarios where sharing the entire app to the meeting stage isn't as useful as sharing specific parts of the app:  
+
+1. For a brainstorming or whiteboard app, a user may want to share a specific board in a meeting vs the entire app with all the boards.  
+
+1. For a medical app, a doctor may want to share just the X-Ray on the screen with the patient vs sharing the entire app with all the patients' records or results etc.  
+
+1. For an app like Arcade, the user would want to share individual games to the meeting stage vs the entire Arcade app.  
+
+1. A user can want to share content from a single content provider at a time (for example, YouTube, TikTok, Disney, etc) vs sharing the entire an entire video catalog onto stage.
+
+To solve for these scenarios, we released APIs within the Teams Client SDK that allows you to programmatically invoke share to stage for specific parts of the app from a button in the app side panel.
+
+:::image type="content" source="../assets/images/apps-in-meetings/shared-meeting-stage-edit-review-component.png" alt-text="The screenshot shows the share to meeting stage view.":::
+
+The following three APIs that are particularly enabling the above scenarios:
+
+|Method| Description| Source|
+|---|---|----|
+|[**Share app content to stage**](#share-app-content-to-stage-api)| Share specific parts of the app to meeting stage from the app side panel in a meeting. | [MSTC SDK](/javascript/api/@microsoft/teams-js/meeting) |
+|[**Get app content stage sharing state**](#get-app-content-stage-sharing-state-api)| Fetch information about app's sharing state on the meeting stage. | [MSTC SDK](/javascript/api/@microsoft/teams-js/meeting.iappcontentstagesharingstate) |
+|[**Get app content stage sharing capabilities**](#get-app-content-stage-sharing-capabilities-api)| Fetch the app's capabilities for sharing to the meeting stage. | [MSTC SDK](/javascript/api/@microsoft/teams-js/meeting.iappcontentstagesharingcapabilities) |
+
+## Share app content to stage API
+
+The `shareAppContentToStage` API enables you to share specific parts of your app to the meeting stage. The API is available through the Teams client SDK.
+
+### Prerequisite
+
+* To use the `shareAppContentToStage` API, you must obtain the RSC permissions. In the app manifest, configure the `authorization` property, and the `name` and `type` in the `resourceSpecific` field. For example:
+
+    ```json
+    "authorization": {
+        "permissions": { 
+        "resourceSpecific": [
+        { 
+        "name": "MeetingStage.Write.Chat",
+        "type": "Delegated"
```

Comment:
nit: your indentation on these two lines is off by one indent

Reply:
@v-sdhakshina please fix this